File: src/Domain/Entities/Product.cs
namespace MargunStore.Domain.Entities
{
    public class Product : EntityBase
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public string Length { get; set; }
        //public string Image { get; set; }
        public decimal Value { get; set; }
        public int Quantity { get; set; }
        public bool Active { get; set; }
        public int CategoryId { get; set; }
        public Category Category { get; set; }
    }
}

File: client/src/app/Models/Product.ts
import { Category } from "./Category";
import { Image } from "./Image";

export interface Product{
    id: number;
    name: string;
    description: string;
    length: string;
    value: number;
    quantity: number;
    active: boolean;
    categoryId: number;
    category: Category;
    images: Image[];
}